code/datums/mutations/hulk.dm

```diff
@@ -112,10 +115,3 @@
 	owner.dna.species.no_equip.Remove(SLOT_WEAR_SUIT, SLOT_W_UNIFORM)
 	owner.physiology.stamina_mod = initial(owner.physiology.stamina_mod)
 	owner.update_body_parts()
-
-/datum/mutation/human/active_hulk/proc/handle_speech(original_message, wrapped_message)
```

Comment:
Where are you moving this proc to? Or is it unused and you're removing?

Reply:
it was removed because i was under the impression it was inheriting the behavior from base hulk and it was working when i would hulk myself with an active hulk injector
![image](https://user-images.githubusercontent.com/46236974/157593855-c7df1fbb-87b0-4899-91b9-e7ba85e66b94.png)